src/types/Review.ts

```diff
@@ -19,7 +20,7 @@ export type Review = {
   isPublished: boolean
   isPending: boolean
```

Comment:
I wasn't sure if these have other envisioned needs in mind, even though they're not currently being utilized. My inclination is to remove them.

Reply:
I agree, lets remove.